doc/classes/TouchScreenButton.xml

```diff
@@ -30,7 +30,10 @@
 			The button's texture for the normal state.
 		</member>
 		<member name="passby_press" type="bool" setter="set_passby_press" getter="is_passby_press_enabled" default="false">
-			If [code]true[/code], pass-by presses are enabled.
+			If [code]true[/code], the [code]pressed[/code] and [code]released[/code] signals are emitted
+			whenever a pressed finger goes in and out of the button,
+			even if the pressure started outside the active area of the button.
```

Comment:
Don't add new lines to the middle of phrases

Reply:
done